chain/jsonrpc/src/lib.rs

```diff
@@ -753,20 +752,6 @@ impl JsonRpcHandler {
         }
     }
 
-    async fn adv_check_refmap(&self, _params: Option<Value>) -> Result<Value, RpcError> {
```

Comment:
why is this removed?

Reply:
Because Block Refmap validation is moved into Store Validator as one of many ways to check Storage Consistency. Running Store Validator you run checking Block Refmap as well.